app/config/AppConfig.scala

```diff
@@ -50,15 +52,26 @@ trait AppConfig {
   def apiGatewayContext: String
   def confidenceLevelConfig: ConfidenceLevelConfig
   def apiStatus(version: Version): String
+
   def featureSwitches: Configuration
+  def endpointsEnabled(version: String): Boolean
   def endpointsEnabled(version: Version): Boolean
```

Comment:
we use to have Version Trait, but now it is back to String. Does this needs to be fixed?

Reply:
It's overloaded, the first is used exclusively by the templates